Autoinst #9216 ke (#827)
SUSE/doc-susemanager: modules/client-configuration/pages/client-automating-installation.adoc

[[client-cfg-autoinstallation-autoyast]]

> == Start an Autoinstallation with {ay}

Reviewer: ```suggestion
== Autoinstallation with {ay}
```

> To start an autoinstallation the system must be known to {productname} already.

Reviewer: ```suggestion
To start an autoinstallation the system must already be known to {productname}.
```

> You can use Bare-metal Provisioning to bring systems into {productname}.

Reviewer: ```suggestion
You can use bare metal provisioning to bring systems into {productname}.
```

For more information, see xref:reference:admin/bare-metal-systems.adoc[].

On the command line, you can use the API calls [systemitem]``system.createSystemRecord`` or [systemitem]``system.createSystemProfile`` to bring systems into {productname}.
For example (replace [literal]``<hw_addr>`` with a hardware address such as [literal]``00:25:22:71:e7:c6`` and [literal]``<name>`` with the name of your system):

----
spacecmd api -- --args '["systemname", {"hwAddress": "<hw_addr>", "hostname":"<name>"}]' system.createSystemProfile
----

> .Procedure

Reviewer: ```suggestion
.Procedure: Autoinstalling with {ay}
```

> . Navigate to menu:Systems[Overview], and select the system for the automated installation.

Reviewer: ```suggestion
. In {productname} {webui}, navigate to menu:Systems[Overview], and select the system for the automated installation.
```

> . On the systems details page, click menu:Provisioning[Autoinstallation > Schedule] and select the {ay} profile you want to use to (re-)install the system.

Reviewer: ```suggestion
. On the systems details page, navigate to the menu:Provisioning[Autoinstallation > Schedule] tab, and select the {ay} profile you want to use to (re-)install the system.
```

> 

Reviewer: ```suggestion
```

=== Advanced PXE Installation Configuration

> The btn:[Create PXE installation configuration] option can be used if the system should be installed for the first time.

Reviewer: ```suggestion
If the client needs to be installed for the first time, you can use the guimenu``Create PXE installation configuration`` option.
```

> A PXE boot configuration will be created.

Reviewer: ```suggestion
This option creates a PXE boot configuration.
```

> When you power on the client system it is booting from the network and the correct profile will be selected for installation.

Reviewer: ```suggestion
When you power on the client system, it boots from the network and the correct profile is selected for installation.
```

> If the system is already managed you can click btn:[Schedule Autoinstallation and Finish] to start the installation.

Reviewer: ```suggestion
If the system is already managed, click btn:[Schedule Autoinstallation and Finish] to start the installation.
```
